### PR TITLE
Add sounds to sportsball and basketball

### PR DESCRIPTION
**Pull request by balderdash**
We have different sets of sounds for traditional bounce, basketball, and sportsball, so I pushed the config from bounce.js and blocks.js into skins.js. Also added the playSound block to the toolbox in the later basketball/sportsball levels.

Also, the hand selector is broken in sportsball right now because of missing hand asset, copied that from basketball.

### CODE REVIEW

**Comment by Hamms** [comment]
~~This PR should be opened against `staging-next`~~

As established offline, this is intended to go into staging for HoC

File: apps/src/bounce/skins.js [hunk 1]
```diff
@@ -238,34 +338,14 @@ exports.load = function (assetUrl, id) {
...
-  skin.retroSound = [skin.assetUrl('2_paddle_bounce.mp3'),
-                     skin.assetUrl('2_paddle_bounce.ogg')];
-  skin.slapSound = [skin.assetUrl('1_wall_bounce.mp3'),
-                    skin.assetUrl('1_wall_bounce.ogg')];
-  skin.hitSound = [skin.assetUrl('2_wall_bounce.mp3'),
-                   skin.assetUrl('2_wall_bounce.ogg')];
+  skin.customSounds = config.customSounds || {};
+  for (var sound in skin.customSounds) {
+    skin.customSounds[sound].urls =
+      skin.customSounds[sound].filenames.map(skin.assetUrl);
```
Comment (Hamms) [inline apps/src/bounce/skins.js:352]: dang, that's pretty

**Comment by balderdash** [comment]
Two more basketball/sportsball changes in this PR:
* Added sharing thumbnails for basketball and sportsball that aren't just a screenshot of bounce
* Added a third hand color

**Comment by balderdash** [comment]
(Waiting until Tuesday's HOC triage to merge)

**Pull request merged**